templates/miq-template.yaml

```diff
@@ -44,12 +44,28 @@ objects:
 - apiVersion: v1
   kind: ImageStream
   metadata:
-    name: miq-images
+    name: miq-app
     annotations:
       description: "Keeps track of the ManageIQ image changes"
   spec:
     dockerImageRepository: "${IMAGES_REGISTRY}/${APPLICATION_IMG_NAME}"
```

Comment:
a nit,

in proper English I think it should be IMAGE_REGISTRY and not IMAGES_REGISTRY